others/career-development.md

```diff
@@ -0,0 +1,73 @@
+# Career Development
+
+## CV
+
+### Best practice
+
+- Keep it short and concise, 1-2 pages maximum
+- Put your details at the top. Only these details are necessary:
+  - Name
+  - Email
+  - Phone number
+  - Your city (not your address since this is personal information! If they need it they'll ask for it later)
+- Use the following sections, and only populate them with relevant details starting from the most recent first
+  1. Education
+  2. Employment
+     - Use action verbs for each bullet point (see: <https://www.indeed.com/career-advice/resumes-cover-letters/action-verbs-to-make-your-resume-stand-out>)
+     - Show your accomplishments, with numbers if you can (e.g. Improved landing page time to interactive by 50%)
+  3. Skills
+  4. Personal projects
+  5. Personal interests (optional)
+
+### Resources
+
+Guide to writing a CV
+
+- <https://www.freecodecamp.org/news/writing-a-killer-software-engineering-resume-b11c91ef699d/>
+
+CV sample
+
+- <https://careercup.com/resume>
```

Comment:
Again this is a résumé, not a CV. Not mentioning which technologies you've been working with role by role in a CV is non-standard to the point I'd suggest it's a bad idea.

Reply:
https://www.livecareer.co.uk/cv-search/r/front-end-developer-00fdd3a63d27437d899f94a70df420be is a pretty decent looking front end dev CV. I'd put skills at the top to make recruiter life easier. The bullet points could use a little work sadly. They're fine but unexciting - mercifully there are no typos. There's the usual issue for our students that it's the generic degree->work flow.

https://www.livecareer.co.uk/cv-search/r/front-end-web-developer-31839afbd6b640429d637cdda953561c is interesting because it's someone self-taught who shows off their portfolio in lieu of qualifications, though they only have one role to display and the formatting is a bit meh generally.

Reply:
I'll update the sample links with yours